src/components/Map/BrowserMap.tsx

```diff
@@ -49,7 +49,13 @@ const BrowserMap = ({ onMapLoaded }) => {
   useUpdateMap(map, viewForMap);
   useUpdateStyle(map, activeLayers);
 
-  return <div ref={mapRef} style={{ height: '100%', width: '100%' }} />;
+  return (
+    <div
+      ref={mapRef}
+      className="bg-white dark:bg-zinc-800"
```

Comment:
I think background should be set more upwards. The browserMap is lazyloaded.

We cant do it on body, because it is in _document. But perhaps the div with className={currentTheme} could be a good candidate for "our body"?